versionconfig.ini

```diff
@@ -1,5 +1,5 @@
 [Version]
-8.6
+8.7
```

Comment:
You no longer need to update this manually. The GitHub workflow actually overwrites it. See the Deployments.MD file.